cppForSwig/Signer/Transactions.cpp

```diff
@@ -21,7 +21,7 @@ TransactionStub::~TransactionStub(void)
 //// TransactionVerifier
 ////////////////////////////////////////////////////////////////////////////////
 ////////////////////////////////////////////////////////////////////////////////
-bool TransactionVerifier::verify(bool noCatch, bool strict) const
+bool Armory::Signer::TransactionVerifier::verify(bool noCatch, bool strict) const
```

Comment:
use namespace Armory::Signer as a one liner solution

Reply:
no, VS complains about ambiguous names if not added explicitly